src/components/ScholarshipDetailCard.jsx

```diff
@@ -1,29 +1,129 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Box, Link, makeStyles, Typography } from '@material-ui/core';
+import {
+  Button,
+  Box,
+  Link,
+  makeStyles,
+  Typography,
+  Chip,
+  Grid,
+  Divider,
+} from '@material-ui/core';
+import { Share, Send, Info } from '@material-ui/icons';
 import ScholarshipAmount from '../types/ScholarshipAmount';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
+  actionsWrapper: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(4),
+    padding: `${theme.spacing(1)}px 0`,
+  },
+  applyBtn: {
+    marginRight: theme.spacing(1),
+  },
   description: {
-    whiteSpace: 'pre-line',
+    color: 'rgba(0, 0, 0, 0.6)',
+    marginBottom: theme.spacing(2),
+  },
+  groupCellWithoutHeader: {
+    marginBottom: theme.spacing(2),
+  },
+  sectionHeader: {
+    fontWeight: '500',
+    color: 'rgba(0, 0, 0, 0.6)',
```

Comment:
awkward color (with the alpha value). Prefer using the non-transparent gray value (it's probably something like 0.4 * 256 = 102.4, so ~100 for each of the rgb values).